desktop_object.py

```diff
@@ -131,6 +127,14 @@ def _get_names(self):
                                               self._localized_names[locale])
         return name_string
 
+    def _get_categories(self):
+        category = self._data['linkCategory']
+        if category == 'Education and Health':
+            category = 'EducationAndHealth'
+        # Start with a valid category per the freedesktop spec,
```

Comment:
I see; if we want to make a change, I would remove `Network` entirely then, since there's no reason to prepend a spec category that I know of, and I would like to avoid this script adding extra data that is not in the CMS when possible.